pydicom/tests/test_encaps.py

```diff
@@ -1199,3 +1199,40 @@ def test_encapsulate_bot(self):
             b'\xfe\xff\x00\xe0'  # Next item tag
             b'\xe6\x0e\x00\x00'  # Next item length
         )
+
+    def test_encapsulate_large(self):
+        """Test encapsulating a large frame."""
+        class FakeBytes(bytes):
```

Comment:
Had to do this because I was getting `MemoryError` with the actions. Also it was a really slow test.